src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
That list seems to be missing FeatureSIMD config value -- something that i happened to notice at a glance.  You might want to double check.